pkg/config/default-config.yml

```diff
@@ -65,8 +64,8 @@ defaultEnvironment:
   MEDIA_DATABASE_PORT: 5432
   MEDIA_DATABASE_NAME: openslides
   MEDIA_DATABASE_USER: openslides
-  MEDIA_DATABASE_PASSWORD: openslides
-  MEDIA_DATABASE_PASSWORD_FILE: /run/secrets/media_postgres_password
+  MEDIA_DATABASE_USER_FILE: /run/secrets/postgres_user
```

Comment:
media-service will fallback to `CONFIG` if it cant find `CONFIG_FILE`, see [here](https://github.com/OpenSlides/openslides-media-service/blob/8435aca151c7529542ee5e9ccbee7aa2f2201f3b/src/config_handling.py#L71)
So for uniformity we should only keep `MEDIA_DATABASE_USER`.